common/scripts/updateVersions.sh

```diff
@@ -1,49 +0,0 @@
-#!/bin/bash -e
```

Comment:
Do we have something similar in place that can replace versions in all of the samples?

As far as I saw this is critical for maven configuration in said samples, since bumping the version would be done by hand in each test.

Reply:
No, we don't. We _could_, but it's not strictly required: the tests pass properties which override whatever is used in the POM file. So the important bit is that the properties are defined, their actual value isn't much important. I can write a task to do the update in any case.

Reply:
Having such task would be nice since I wouldn't like having `samples` that point to a older release in the master. I know that it would confuse me as a end user (since I would just copy a sample and build upon that).

Reply:
I'd be keen on doing this in a separate PR later, since it will require some more rework which would make this PR even longer, wdyt?

Reply:
Sounds good to me.

Reply:
I actually ended up doing it in this PR, since it involved less work than I initially thought.